lib/utilities.bash

```diff
@@ -60,15 +60,21 @@ function _bash-it-array-dedup() {
 	printf '%s\n' "$@" | sort -u
 }
 
-# Outputs a full path of the grep found on the filesystem
+# Runs `grep` with *just* the provided arguments
 function _bash-it-grep() {
-	: "${BASH_IT_GREP:=$(type -P egrep || type -P grep)}"
-	printf "%s" "${BASH_IT_GREP:-/usr/bin/grep}"
+	: "${BASH_IT_GREP:=$(type -P grep)}"
+	"${BASH_IT_GREP:-/usr/bin/grep}" "$@"
 }
 
-# Runs `grep` with extended regular expressions
+# Runs `grep` with fixed-string expressions (-F)
+function _bash-it-fgrep() {
+	: "${BASH_IT_GREP:=$(type -P grep)}"
+	"${BASH_IT_GREP:-/usr/bin/grep}" -F "$@"
+}
```

Comment:
I considered trying to move the `BASH_IT_GREP` assignment outside of the functions so that we could only have to execute it once, but wasn't sure _where_ the best place for that would be.